typescript/deploy-sdk/src/AltVMCoreModule.ts

```diff
@@ -195,11 +185,11 @@ export class AltVMCoreModule
 
     if (config.defaultHook && typeof config.defaultHook !== 'string') {
       switch (config.defaultHook.type) {
-        case HookType.INTERCHAIN_GAS_PAYMASTER: {
+        case 'interchainGasPaymaster': {
```

Comment:
how come we are using strings here? can't we use HookType defined below in `typescript/sdk/src/hook/types.ts`?

Reply:
i suppose we can potentially do that later